env.d/grafana

```diff
@@ -24,4 +24,4 @@ GF_DEFAULT_APP_MODE="development"
 
 # Plugins
 GF_PATHS_PLUGINS=/var/lib/grafana/plugins/
-GF_PLUGINS_ALLOW_LOADING_UNSIGNED_PLUGINS=grafana-groupedbarchart-panel
+GF_PLUGINS_ALLOW_LOADING_UNSIGNED_PLUGINS=potsie-stackedbarchart-panel
```

Comment:
```suggestion
GF_PLUGINS_ALLOW_LOADING_UNSIGNED_PLUGINS=potsie-stacked-barchart-panel
```

Reply:
Good Idea) Thanks)

Reply:
Aye, the json schema in `plugin.json` complains now about the plugin name: `String does not match the pattern of "^[0-9a-z]+\-([0-9a-z]+\-)?(app|panel|datasource)$"`, I re-renamed it back to `potsie-stackedbarchart-panel`(